contracts/tgrade-validator-voting/src/contract.rs

```diff
@@ -187,10 +187,15 @@ pub fn execute_execute(
             code_id,
             migrate_msg,
         } => {
-            res = res.add_message(WasmMsg::Migrate {
-                contract_addr: contract,
-                new_code_id: code_id,
-                msg: migrate_msg,
+            res = res.add_message(TgradeMsg::ExecuteGovProposal {
```

Comment:
I understand what this change does, but I am not sure the why.

I believe you argued against giving more power than necessary to the validators and just giving them the right to update the core poe-contracts and not all the various DeFi contracts running on the chain by anyway. You had convinced me that is the correct path, and the validators should not be able to interfere with some project a bank is running on our chain.